src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AgencyPrisonerPayProfileIntTest.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.HttpMethod
+import uk.gov.justice.hmpps.prison.api.model.AgencyPrisonerPayProfile
+import uk.gov.justice.hmpps.prison.api.model.ErrorResponse
+import java.math.BigDecimal
+import java.time.LocalDate
+
+class AgencyPrisonerPayProfileIntTest : ResourceTest() {
+  @Test
+  fun agencyPayProfile_returnsSuccessAndData() {
+    val requestEntity = createHttpEntityWithBearerAuthorisation("ITAG_USER", listOf("ROLE_VIEW_PRISON_DATA"), emptyMap())
+    val responseEntity = testRestTemplate.exchange(
```

Comment:
for future reference new tests should be using webclient, not the old rest template

Reply:
Noted, thanks.